README.md

```diff
@@ -1,19 +1,21 @@
 [![codecov](https://codecov.io/gh/101Loop/APIManager-Flutter/branch/master/graph/badge.svg?token=770R0EZKQG)](undefined)
+## ABOUT PROJECT
 
-## APIManager-Flutter
+# APIManager-Flutter
 `APIManager-Flutter` is an API manager for flutter applications that can manage
 the API calls from a single place.<br>
 
 # 🔮 What's an API?
   API is the acronym for Application Programming Interface, which is a software intermediary that allows two applications to talk to each other.<br> 
 <br>  When you use an application on your mobile phone, the application connects to the Internet and sends data to a server. The server then retrieves that data, interprets it, performs the necessary actions and sends it back to your phone. The application then interprets that data and presents you with the information you wanted in a readable way. This is what an API is - all of this happens via API.<br>
-<br><br> 
 
-## 💡 What's Unique?
+
+# 💡 What's Unique?
 <br> The handling of APIs is usually messy. So here we present to you a much simplified way to beautifully handle the API calls.
 
-## Features
-- [x] Manage token 
+# Features
+ Here's the list of your all-in-one service
+- [ ] Manage token 
```

Comment:
@nayakastha you have removed the mark from her by mistake... plz revert it and you're good to go

Reply:
I am so sorry. I am doing it right away